docker-compose.yml

```diff
@@ -56,6 +58,7 @@ services:
       - ${PMM_PORT_CH_HTTP:-11123}:8123
     volumes:
       - ./:/root/go/src/github.com/percona/pmm
+      - ../../percona-platform/grafana/public:/usr/share/grafana/public
```

Comment:
That may be a problem for other devs. Do we need it, or that left after debugging?

Reply:
Yeah, it's committed by mistake